sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MainActivity.java

```diff
@@ -206,7 +205,7 @@ protected void onResume() {
     final ISpan span = Sentry.getSpan();
     if (span != null) {
       span.setMeasurement("screen_load_count", screenLoadCount, new MeasurementUnit.Custom("test"));
-      span.finish(SpanStatus.OK);
+      // span.finish(SpanStatus.OK);
```

Comment:
@romtsn I found that using `Sentry.getSpan()` and then `span.finish()` is finishing the ttid span
Is it something to be worried about? Would a user be already using something like this?

Reply:
yes, it's possible that the users already do this to finish the activity transaction whenever they think it's suitable for them. 

Once we implement https://github.com/getsentry/sentry-java/issues/1828, this should not matter anymore, but until then, we should probably mention this in the docs + migration docs, that calling span.finish in onResume will change how ttid span is measured.